mod-record-specifications-server/src/main/java/org/folio/rspec/domain/entity/support/Metadata.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.rspec.domain.entity.support;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import java.sql.Timestamp;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Getter
+@Setter
+@Embeddable
+public class Metadata {
+
+  public static final String CREATED_DATE_COLUMN = "created_date";
+  public static final String UPDATED_DATE_COLUMN = "updated_date";
+  public static final String CREATED_BY_USER_COLUMN = "created_by_user_id";
+  public static final String UPDATED_BY_USER_COLUMN = "updated_by_user_id";
```

Comment:
Is it added to use this constants in many places? Name mappings should work out of the box with spring boot using CamelCaseToUnderscoresNamingStrategy by default.

Reply:
It's just for reference in index declarations (check Specification entity) or in JPA spec queries.